src/common/config_parser.cpp

```diff
@@ -103,7 +108,7 @@ ConfigParser::ConfigParser(cli::mode mode)
   // clang-format on
 }
 
-void ConfigParser::addOptionsGeneral(cli::CLIWrapper& cli) {
+void ConfigParser::addOptionsGeneral(cli::CLIWrapper & cli) {
```

Comment:
Minor: let's skip unnecessary changes so that merging other PRs will be potentially slightly easier.